static/sass/_pattern_p-side-navigation.scss

```diff
@@ -0,0 +1,27 @@
+@mixin p-charmhub-side-navigation {
+  $color-sidenav-item-background-highlight: #cef3ff;
+  $color-sidenav-item-border-highlight: #35cdff;
+
+  .p-side-navigation__item {
+    .p-side-navigation__link {
+      .p-side-navigation__status {
+        display: none;
+      }
+
+      &:hover,
+      &.is-active {
+        background: $color-sidenav-item-background-highlight;
+      }
+
+      &:hover {
+        .p-side-navigation__status {
+          display: block;
+        }
+      }
+
+      &.is-active {
+        @include vf-highlight-bar($color-sidenav-item-border-highlight, left);
```

Comment:
You should need to do this. This highlight should use the built in accent color. https://github.com/canonical-web-and-design/vanilla-framework/issues/2966